portal/home.php

```diff
@@ -210,17 +216,14 @@ function saveProfile(){
 					<div class="panel panel-primary collapse" id="downloadpanel">
 						<header class="panel-heading"> <?php echo xlt('Download Documents'); ?> </header>
 						<div id="docsdownload" class="panel-body">
-						<?php if ( $GLOBALS['portal_onsite_document_download'] ) { ?>
+						<?php if ( $GLOBALS['portal_onsite_document_download'] ) { ?>// really should always be able to download!
```

Comment:
Are you suggesting that patients actually should own their own medical data?(sarcasm)  :)